cmd/marker/main.go

```diff
@@ -16,27 +16,38 @@ package main
 
 import (
 	"flag"
-	"time"
-
+	"github.com/Rican7/retry/jitter"
```

Comment:
Let's use Jitter function from Kubernetes' API Machinery (which is already shipped in the project I believe) instead. https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#Jitter